<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>241</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>ВВОД</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="idLineEdit">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>70</y>
      <width>221</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="titleLineEdit">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>140</y>
      <width>221</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="genreLineEdit">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>210</y>
      <width>221</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ratingLineEdit">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>280</y>
      <width>221</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="isAvailableCheckBox">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>320</y>
      <width>123</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Доступен</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>40</y>
      <width>92</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>id</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>110</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>название</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>180</y>
      <width>92</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>рейтинг</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>250</y>
      <width>92</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>жанр</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>391</width>
      <height>89</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>729</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
